app/views/customers/new.html.slim

```diff
@@ -33,7 +31,13 @@
                     br
                     small Step 3 Tempoary Address
             .stepContainer
-              = form_for @customer, url: customer_index_path do |f|
+              = form_for @customer, url: customers_path  do |f|
```

Comment:
@RaviMariya-BTC If some how customer failed to save, Then the page will render 'new', Where you will show the cause for the break of code or we can say validation or any other problem.